recipes/fsleyes/yum_requirements.txt

```diff
@@ -0,0 +1,2 @@
+gtk2
+xorg-x11-server-Xvfb
```

Comment:
Please don't rely on yum_requirements. See https://github.com/conda-forge/conda-forge.github.io/pull/690

Reply:
Well since the wxpython recipe no longer uses `yum_requirements.txt`, it looks like I no longer need to either.